tutorials/plot_bem.py

```diff
@@ -23,23 +25,31 @@
     )
 
 ########################################################
-# First, create the required anatomical files
-
-hcp.make_mne_anatomy(**anatomy_params)
+# This function assumes that we have created the required anatomical files.
+# A simple way for doing that is using the function hcp.make_mne_anatomy;
+# it takes similar parameters as the hcp.compute_forward_stack function
+# we will be using next, so you could just run hcp.make_mne_anatomy(**anatomy_params)
+# to set up the anatomy files.
 
-########################################################
-# Next, compute the 'forward stack':
+# But for now, we first, compute the 'forward stack':
 # With one handy function, construct everything required for inverse modelling.
 # Results are stored in a dict.
 # Note this step can take up to a few minutes;
-# particularly calculating the forward model is computationally expensive.
+# particularly calculating the forward model/morphing is computationally expensive.
 
 forward_results = hcp.compute_forward_stack(**anatomy_params)
 print(forward_results.keys())
 
 ########################################################
-# Now we can plot the BEM solution.
-# We are using a single-layer BEM because HCP only has MEG data.
+# We can plot the BEM solution. We are using a single-layer BEM because HCP only has MEG data.
 
-bem = out["bem_sol"]
 mne.viz.plot_bem(subject=subject, subjects_dir=subjects_dir)
+
+########################################################
+# We can also plot the sensitivity map based on the forward model.
+# (This 3D plot requires the Python2 packge Mayavi to be installed.)
+
+fwd = forward_results["fwd"]
+grad_map = mne.sensitivity_map(fwd, ch_type='mag', mode='fixed')
+grad_map.plot(time_label='Sensitivity', subjects_dir=subjects_dir + 'T1w/',
```

Comment:
Wrong directory. You point to hcp_path, it should be subjects_dir ( the one that has been created by linking with make_mne_anatomy).